exercises/I-array-properties/exercise.js

```diff
@@ -6,8 +6,11 @@
 */
 
 function isEmpty(arr) {
-  return; // complete this statement
-}
+  if (arr[0] == undefined) {
+    return true; // complete this statement
+} else {
+  return false
+}}
```

Comment:
This could fail sometimes - for example it would class the array `[undefined, 'Erin', 'Paul']` as empty. You could check the length as an alternative